### ISSUE TITLE
JoyRide Missing Configuration Options

### ISSUE THREAD

**Issue by CautemocSg** [pineconellc/angular-foundation#290]
According to http://foundation.zurb.com/sites/docs/v/5.5.3/components/joyride.html there are a large number of configuration options that are not exposed in this. Most importantly, the callbacks seem to not be available. Pre-ride, post-step, and post-ride callbacks are vital for walking a user through the view. Is there any way to utilize those callbacks using angular-foundation?

**Comment by jbrowning** [comment]
No, unfortunately we don't have this sort of functionality built in. Our tour components work differently and since our tour behaves more implicitly, it would be a little awkward to add something like this in. I'd be happy to take a look at a PR/proposal to add in some sort of callback functionality.
